src/main/resources/db/migration/oracle/V1.3__create_table_for_texts.sql

```diff
@@ -0,0 +1,7 @@
+create table texts
+(
+    key          varchar2(255) not null,
```

Comment:
Foreslår kode, verdi, sprak, fra_dato

Reply:
kode, verdi, sprak, fra_tidspunkt

Reply:
Hvorfor tidspunkt?

Reply:
Fordi det er tidspunkt. Dato og tid

Reply:
For å få muligheten til å være mer presis.

Reply:
hm, forstår ikke når det kan være et behov, men vi kan diskutere det på møtet senere i dag.